### ISSUE TITLE
[ExposedDropdownMenuBox] DropdownMenuItem items announced as "unlabelled" by TalkBack

### ISSUE THREAD

**Issue by alp-well** [material-components/material-components-android#5012]
When using ExposedDropdownMenuBox with DropdownMenuItem, TalkBack announces menu items as "unlabelled" instead of reading the item text.

Expected behavior: TalkBack should announce the item text (e.g. "Cranberry") when focusing on a DropdownMenuItem.

Source code:

  ExposedDropdownMenuBox(
      expanded = expanded,
      onExpandedChange = { expanded = it }
  ) {
      OutlinedTextField(
          modifier = Modifier.menuAnchor(ExposedDropdownMenuAnchorType.PrimaryNotEditable),
          value = selectedItem,
          onValueChange = {},
          readOnly = true,
          label = { Text("Select a fruit") },
          trailingIcon = { ExposedDropdownMenuDefaults.TrailingIcon(expanded = expanded) }
      )

      DropdownMenu(
          modifier = Modifier.exposedDropdownSize(),
          expanded = expanded,
          onDismissRequest = { expanded = false }
      ) {
          items.forEach { item ->
              DropdownMenuItem(
                  text = { Text(item) },
                  onClick = {
                      selectedItem = item
                      expanded = false
                  },
                  contentPadding = ExposedDropdownMenuDefaults.ItemContentPadding
              )
          }
      }
  }

Minimal sample app repro: https://github.com/alpertign/ExposedDropdownMenuBoxIssueA

Android API version: Android 14

Material Library version: BOM 2024.12.01 (Material3 1.3.1)

Device: Xiaomi  (with TalkBack enabled)

**Comment by alpertign** [comment]
https://github.com/user-attachments/assets/10f6d881-5220-4afa-a160-0ed330f630e5

**Comment by drchen** [comment]
Hey Dan, this seems a11y-related. Can you help triage this?

**Comment by dsn5ft** [comment]
Hi, this looks like a Compose issue, please file at https://issuetracker.google.com/issues/new?component=742043&template=1346811 instead (this repo is for Material Android Views only).

**Comment by alp-well** [comment]
Created [this](https://issuetracker.google.com/issues/493304623)
